src/layer/marker/Marker.Label.js

```diff
@@ -0,0 +1,9 @@
+/*
+ * Label extension to L.Marker, adding label-related methods.
+ */
+
+L.Marker.include({
+	_getLabelAnchor: function () {
+		return this.options.icon.options.labelAnchor || [0, 0];
+	}
+});

```

Comment:
Maybe let's just move this to `Marker.js` — this is too small to be a separate file.